ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -315,35 +228,40 @@ S3FileOpen(WT_FILE_SYSTEM *fileSystem, WT_SESSION *session, const char *name,
         return (EINVAL);
     }
 
+    // Check if the file exists and get the size
+    bool objExists;
+    wt_off_t objSize;
+    if ((ret = S3FileExistsAndGetSize(fileSystem, session, name, &objExists, &objSize)) != 0) {
+        s3->log->LogErrorMessage("S3FileOpen: error checking if the file exists. Ret code:" + ret);
```

Comment:
`strerror` is a more useful tool here - the numeric code can change between systems so it's better to print some text.

Reply:
Thank you changed here [12ff933](https://github.com/wiredtiger/wiredtiger/pull/9708/commits/12ff9339c1e48e451f1f053627295e0c4ddb4836)

Reply:
I still have a nitpick with the new code sorry. There should be a space before the string, and it's no longer a code getting printed, so it would be a little clearer without that bit of the text (i.e. `S3FileOpen: error checking if the file exists: " + std::string(...)`